spec/features/users_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :feature do
```

Comment:
feature のテストは以下のようになります（Capybara の構文）

```suggestion
RSpec.feature '会員登録', type: :feature do
```